fire/cli/niv/_regn.py

```diff
@@ -145,28 +147,20 @@ def spredning(
 
 
 # ------------------------------------------------------------------------------
-def find_fastholdte(punktoversigt: pd.DataFrame, kontrol: bool) -> Dict[str, float]:
+def find_fastholdte(punktoversigt: np.ndarray, kontrol: bool) -> Dict[str, float]:
     if kontrol:
-        relevante = punktoversigt[punktoversigt["Fasthold"] == "x"]
+        relevante = punktoversigt[punktoversigt[:,1] == "x"]
     else:
-        relevante = punktoversigt[punktoversigt["Fasthold"] != ""]
+        relevante = punktoversigt[punktoversigt[:,1] != ""]
 
-    fastholdte_punkter = tuple(relevante["Punkt"])
-    fastholdteKoter = tuple(relevante["Kote"])
+    fastholdte_punkter = tuple(relevante[:,0])
+    fastholdteKoter = tuple(relevante[:,3])
     return dict(zip(fastholdte_punkter, fastholdteKoter))
 
-
-# ------------------------------------------------------------------------------
-def gama_beregning(
-    projektnavn: str,
-    observationer: pd.DataFrame,
-    arbejdssæt: pd.DataFrame,
-    estimerede_punkter: Tuple[str, ...],
-    kontrol: bool,
-) -> Tuple[pd.DataFrame, str]:
-    fastholdte = find_fastholdte(arbejdssæt, kontrol)
-
-    # Skriv Gama-inputfil i XML-format
+def skriv_gama(projektnavn: str, fastholdte: dict, estimerede_punkter:  Tuple[str, ...], observationer: np.ndarray):
+    """
+    Skriv gama-inputfil i XML-format
```

Comment:
Udvid gerne docstring med en beskrivelse af indholdet i de enkelte input. Især `observationer` ser ud til at have en meget specifik struktur der kan være svær at gennemskue uden noget at støtte sig til